lib/uksched/sched.c

```diff
@@ -47,6 +45,9 @@
 
 struct uk_sched *uk_sched_head;
 
+/* TODO: Define per CPU */
```

Comment:
When should this TODO item be fixed, i.e define `__uk_sched_thread_current` as a per-CPU variable?

Reply:
This is to be done as soon as we add SMP support to the scheduling API library. We need a per-CPU variable declaration. As soon as we have this, multiple non-SMP-aware schedulers (one per CPU) could be executed or a scheduler that can handle multiple cores.

Reply:
I added some more comments to the commit message and to the comment in the source.